log/gelfforwarder.go

```diff
@@ -71,6 +71,7 @@ func (b *gelfBackend) sendMessage(parts *rawLogParts, appName, processName, cont
 			"_pid": processName,
 		},
 		RawExtra: b.extra,
+		TimeUnix: float64(time.Now().UnixNano()) / 1E9,
```

Comment:
Nitpick: Could you please change `1E9` to `time.Second` to make it clearer when reading the code in the future?

Reply:
Done! =)